ScooterTest/ScooterTest.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Scooter;
+using System.Collections.Generic;
+
+namespace ScooterTest
+{
+    [TestClass]
+    public class ScooterTest
+    {
+        [TestMethod]
+        public void Update_And_Retrieve_Scooter_Inventory()
+        {
+            RentalCompany rc = new RentalCompany("Name of the company");
+
+            rc.ScooterService.AddScooter("1", 0.1m);
+            rc.ScooterService.AddScooter("2", 0.1m);
+            rc.ScooterService.AddScooter("3", 0.1m);
+            rc.ScooterService.AddScooter("4", 0.1m);
+
+            rc.StartRent("4");
+            rc.EndRent("4");
+            rc.StartRent("3");
+
+            var expectedInventory = new List<Scooter.Scooter>()
+            {
+                rc.ScooterService.GetScooterById("1"),
+                rc.ScooterService.GetScooterById("2"),
+                rc.ScooterService.GetScooterById("4")
+            };
+
+            CollectionAssert.AreEqual(expectedInventory, rc.ScooterService.GetScooters());
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(Exception), "No scooter found by id: 5")]
+        public void Accessing_Non_Existant_Scooter()
+        {
+            RentalCompany rc = new RentalCompany("Name of the company");
+
+            rc.ScooterService.AddScooter("1", 0.1m);
+            rc.StartRent("5");
+        }
+
+        [TestMethod]
+        public void Removing_Scooter()
+        {
+            RentalCompany rc = new RentalCompany("Name of the company");
```

Comment:
ja testē RentalCompany tad tikai caur interfeisu. :) 
neļaus implementēt un testēt lietas kuras nav interfeisā.. 
bet šis ir ScooterService tests kuram arī jābūt ir atseviķā testa klasē.